scripts/apps/users/directives/UserPreferencesDirective.ts

```diff
@@ -331,12 +333,9 @@ export function UserPreferencesDirective(
                     return $q.when();
                 }
 
-                msg = [
-                    'No preferred categories selected. Should you ',
-                    'choose to proceed with your choice, a default ',
-                    'set of categories will be selected for you.',
-                ].join('');
-                msg = gettext(msg);
+                msg = gettext('No preferred categories selected.') + ' ' +
```

Comment:
Let's not split up the translations artificially. This is a single message, let it be one translation. I would simply put it in one line and tell eslint to ignore the line length.